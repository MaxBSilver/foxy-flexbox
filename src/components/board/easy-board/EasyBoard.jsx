import React from "react";


export default function EasyBoard(props) {
  const direction = props.flexDirection;
  const foxAlign = props.userGuessTwo;
  const boxAlign = props.alignContent;
  const foxJustify = props.userGuessOne;
  const boxJustify = props.justifyContent;

  return (
    <section className="board--container" style={{ flexDirection: direction }}>
      <section
        className="fox--container"
        style={{ justifyContent: foxJustify, alignItems: foxAlign }}
      >
        <article className="fox">
          <img
            className="fox-img"
            alt="fox"
            src="http://clipart-library.com/img/749202.png"
          />
        </article>
      </section>
      <section
        className="box--container"
        style={{ justifyContent: boxJustify, alignItems : boxAlign }}
      >
        <article className="box">
        </article>
      </section>
    </section>
  );
}
