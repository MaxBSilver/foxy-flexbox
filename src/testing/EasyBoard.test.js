import React from 'react';
import EasyBoard from '../components/board/easy-board/EasyBoard';
import { shallow } from 'enzyme';
describe('EasyBoard', () => {
    let wrapper;
  
    beforeEach(() => {
      wrapper = shallow(
        <EasyBoard
        />
      );
    });
    it('should match the snapshot with all data passed in', () => {
        expect(wrapper).toMatchSnapshot();
      });
  });