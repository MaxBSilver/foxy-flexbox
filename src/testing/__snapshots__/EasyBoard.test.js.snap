// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EasyBoard should match the snapshot with all data passed in 1`] = `
<section
  className="board--container"
  style={
    Object {
      "flexDirection": undefined,
    }
  }
>
  <section
    className="fox--container"
    style={
      Object {
        "alignItems": undefined,
        "justifyContent": undefined,
      }
    }
  >
    <article
      className="fox"
    >
      <img
        alt="fox"
        className="fox-img"
        src="https://i.imgur.com/XnHxBnB.png"
      />
    </article>
  </section>
  <section
    className="box--container"
    style={
      Object {
        "alignItems": undefined,
        "flexWrap": undefined,
        "justifyContent": undefined,
      }
    }
  >
    <article
      className="box"
    />
  </section>
</section>
`;
